test/test_remap.py

```diff
@@ -154,9 +154,137 @@ def test_edge_centers_remapping(self):
+            remap_to="nodes",
+            coord_type="cartesian",
+            k_neighbors=1)
+
+        # two different k_neighbor remaps are different
+        assert not np.array_equal(destination_data_neighbors_1,
+                                  destination_data_neighbors_2)
+
+    def test_remap_return_types(self):
+        """Tests the return type of the `UxDataset` and `UxDataArray`
+        implementations of Inverse Distance Weighted."""
+
+        source_data_paths = [
+            dsfile_v1_geoflow, dsfile_v2_geoflow, dsfile_v3_geoflow
+        ]
+        source_uxds = ux.open_mfdataset(gridfile_geoflow, source_data_paths)
+        destination_uxds = ux.open_dataset(gridfile_CSne30,
+                                           dsfile_vortex_CSne30)
+
+        remap_uxda_to_grid = source_uxds['v1'].inverse_distance_weighted_remap(
+            destination_uxds.uxgrid)
+
+        assert isinstance(remap_uxda_to_grid, UxDataArray)
+
+        remap_uxda_to_uxda = source_uxds['v1'].inverse_distance_weighted_remap(
+            destination_uxds['psi'])
+
+        # Dataset with two vars: original "psi" and remapped "v1"
+        assert isinstance(remap_uxda_to_uxda, UxDataset)
+        assert len(remap_uxda_to_uxda.data_vars) == 2
+
+        remap_uxda_to_uxds = source_uxds['v1'].inverse_distance_weighted_remap(
+            destination_uxds)
+
+        # Dataset with two vars: original "psi" and remapped "v1"
+        assert isinstance(remap_uxda_to_uxds, UxDataset)
+        assert len(remap_uxda_to_uxds.data_vars) == 2
+
+        remap_uxds_to_grid = source_uxds.inverse_distance_weighted_remap(
+            destination_uxds.uxgrid)
+
+        # Dataset with three vars: remapped "v1, v2, v3"
+        assert isinstance(remap_uxds_to_grid, UxDataset)
+        assert len(remap_uxds_to_grid.data_vars) == 3
+
+        remap_uxds_to_uxda = source_uxds.inverse_distance_weighted_remap(
+            destination_uxds['psi'])
+
+        # Dataset with four vars: original "psi" and remapped "v1, v2, v3"
+        assert isinstance(remap_uxds_to_uxda, UxDataset)
+        assert len(remap_uxds_to_uxda.data_vars) == 4
+
+        remap_uxds_to_uxds = source_uxds.inverse_distance_weighted_remap(
+            destination_uxds)
+
+        # Dataset with four vars: original "psi" and remapped "v1, v2, v3"
+        assert isinstance(remap_uxds_to_uxds, UxDataset)
+        assert len(remap_uxds_to_uxds.data_vars) == 4
+
+    def test_edge_remapping(self):
+        """Tests the ability to remap on edge centers using Inverse Distance
+        Weighted Remapping."""
+
+        # Open source and destination datasets to remap to
+        source_grid = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
+        destination_grid = ux.open_dataset(mpasfile_QU, mpasfile_QU)
+
+        # Perform remapping to the edge centers of the dataset
+
+        remap_to_edge_centers = source_grid['v1'].inverse_distance_weighted_remap(destination_obj=destination_grid,
```

Comment:
where is the variable v1 located in source grid? (It must also be on edge centers?)

Reply:
It doesn't matter where the variable is located, so you can remap from nodes to edges if you want.